content/systems.md

```diff
@@ -37,29 +37,28 @@ The software configuration will consist of the following:
     - gedit
     - geany
     - kate
-    - atom
 - IDEs:
-    - IntelliJ Community
-        - Version: 2021.3.2
-        - Workspace is already set up
+    - Idea Community
```

Comment:
```suggestion
    - IntelliJ IDEA Community
```

Reply:
I specifically removed that because I found it weird that IDEA is the only editor including the "IntelliJ" brand name, even though PyCharm and CLion are also IntelliJ editors? 😄

Reply:
Not really; they're all editors created by JetBrains, but this editor specifically is branded as IntelliJ IDEA (verbatim captialisation), while the others do not use the term "IntelliJ" :slightly_smiling_face:

Reply:
To answer your observation of the "weird naming patterns", some history on the naming for Jetbrains editors
>The name JetBrains stems from word play on the phrase “thinking fast”. Then there’s the name of one of our first and most important products, IntelliJ IDEA, which is relatively easy to decode. It’s an Intelligent Java IDE(a).
>
> Generally our product names reflect this two-part structure established with names of the company and the first product. One word suggests the technology covered by the product, and another word carries an emotional meaning (e.g. “charm”, “storm”, etc.). Over time, following this rule started to get tough, and we deviated from the pattern with CLion and AppCode. We just gave up altogether with Rider. Kotlin, our programming language, is named after an island near St. Petersburg. JetBrains Space is probably the most generic name of our products and this was intentional, as it is a space for all teams and team members to work together. [source](https://blog.jetbrains.com/space/2020/12/16/space-ama-session/)